archive/s/swift/README.md

```diff
@@ -8,6 +8,7 @@ Welcome to Sample Programs in Swift!
 - [Reverse String in Swift](https://therenegadecoder.com/code/reverse-a-string-in-swift/)
 - [Fizzbuzz in Swift](https://github.com/TheRenegadeCoder/sample-programs/issues/482)
 - [Baklava in Swift](https://github.com/TheRenegadeCoder/sample-programs/issues/620)
+- [Fibonacci String in Swift](https://sample-programs.therenegadecoder.com/projects/fibonacci/)
```

Comment:
```suggestion
- Fibonacci String in Swift
```

I removed the link here. Usually, we link an article or an article issue, but I think it's fine without.